src/app/model/DatasetInfo.ts

```diff
@@ -42,3 +46,18 @@ export interface UploadDatasetError {
   error_code: string;
```

Comment:
Maybe "code" and "message"?

Reply:
It seems our API has been updated to comply with the OpenAPI specification. This means we can now transition to the new API client, which comes with built-in TypeScript types. Since the old API is no longer functional, we cannot verify the accuracy of the referenced interface from the old system, so I propose to update our API client in the next task.